sign_oca/__manifest__.py

```diff
@@ -14,10 +14,10 @@
         "security/security.xml",
         "views/menu.xml",
         "data/data.xml",
-        "wizards/res_config_settings_views.xml",
         "data/ir_sequence_data.xml",
         "wizards/sign_oca_template_generate.xml",
         "wizards/sign_oca_template_generate_multi.xml",
+        "views/res_config_settings_views.xml",
```

Comment:
Because even if they are transient `res.config.settings` they have always been set in models and in views as exception (because they are not really a wizard).

Reply:
I would say that's a mistake, but being in the other place, let's not change it.